index.html

```diff
@@ -1374,15 +1374,39 @@ <h2>Progressive Trust</h2>
       <p>
 Entities that use <a>verifiable credentials</a> and
 <a>verifiable presentations</a> should follow protocols that enable progressive
-trust.
+trust. Progressive trust refers to enabling individuals to share information
+about themselves only on an as needed basis, slowing building up more trust as
+more information is shared with another party.
+      </p>
+      <p>
+Progressive trust is strongly related to the principle of data minimization, and
+enabled by technologies such as selective disclosure and predicate proofs. We
+encourage the use of progressive trust as a guiding principle for implementers
+as they develop protocols for <a>issuers</a>, <a>holders</a>, and
+<a>verifiers</a>.
+
       </p>
 
       <section>
         <h3>Data Minimization</h3>
         <p>
 Data minimization is a principle that encourages <a>verifiers</a> to request the
 minimum amount of data necessary from <a>holders</a>, and for <a>holders</a> to
-only provide the minimum amount of data to <a>verifiers</a>.
+only provide the minimum amount of data to <a>verifiers</a>. This "minimum
+amount of data" depends on the situation and may change over the course of a
+<a>holder</a>'s interaction with a <a>verifier</a>.
+        </p>
+        <p>
+For example, if a <a>holder</a> is applying for a loan, with the bank acting as
+the <a>verifier</a>. There are several points at which the bank may be able to
+determine whether the <a>holder</a> is qualified to continue in the process of
+applying for the loan. A bank may have a policy of only establishing loans with
+existing account holders. A protocol that follows the principle of data
+minimization will allow the <a>holder</a> to only reveal that they are an
+existing account holder, before requiring any more information from the
+applicant. In this way, trust is built between the applicant and the bank
+progressively, as the minimum amount of data needed by the bank to make its
+determinations is requested a piece at a time.
```

Comment:
```
For example, if a <a>holder</a> is applying for a loan, with the bank acting as
the <a>verifier</a>. There are several points at which the bank may be able to
determine whether the <a>holder</a> is qualified to continue in the process of
applying for the loan. A bank may have a policy of only establishing loans with
existing account holders. A protocol that follows the principle of data
minimization will allow the <a>holder</a> to only reveal that they are an
existing account holder, before requiring any more information from the
applicant. In this way, trust is built between the applicant and the bank
progressively, as the minimum amount of data needed by the bank to make its
determinations is requested a piece at a time.
```
->
```
For example, a <a>holder</a> may apply for a loan, with a bank acting as the 
<a>verifier</a>. There are several points at which the bank may want to determine 
whether the <a>holder</a> is qualified to continue in the process of applying for 
the loan; for instance, the bank may have a policy of only providing loans to 
existing account holders. A protocol that follows the principle of data 
minimization would allow the <a>holder</a> to reveal to the <a>verifier</a> only 
that they are an existing account holder, before the bank requests any additional 
information, such as account balances or employment status. In this way, the 
applicant may progressively entrust the bank with more information, as the data 
needed by the bank to make its determinations is requested a piece at a time, as 
needed, rather than as a complete set, up front.
```

Reply:
Thanks for the edits! 
addressed in [a2642eb](https://github.com/w3c/vc-imp-guide/pull/14/commits/a2642eb58336bcc4303270b72f9e3ffcc8cd74bc)